users/userRouter.js

```diff
@@ -1,29 +1,59 @@
-const express = 'express';
-
+const express = require('express')
+const db = require("./userDb");
 const router = express.Router();
 
-router.post('/', (req, res) => {
-
+router.post('/', validateUser , (req, res) => {
+  const userdata = req.body;
+  db.insert(userdata)
+  .then(response => {
+      res.status(200).send(response)
+  })
+  .catch(error => {
+      res.send(error);
```

Comment:
```suggestion
      res.status(500).send(error);
```
It's a good idea to let the user know the status code.